### PR TITLE
Add cURL template. Backbone-ify client-side application.

### PR DESCRIPTION
**Pull request by cianclarke**

### CODE REVIEW

**Comment by cianclarke** [comment]
@lfryc I think this P/R is probably more important than #6 right now. #6 can be reviewed later, this is some stuff needed to finish out Phase 1. 
We should maybe consider unit testing the client side too...

**Comment by lfryc** [comment]
@cianclarke yea, frontend tests may come handy, created https://issues.jboss.org/browse/FH-1961

Tested, works fine. Merging this. :+1:

**Comment by lfryc** [comment]
Landed: https://github.com/feedhenry-templates/fh-api-mapper/commit/1d9fb20d7764b70acbcf11e98fa34f0ddece5d83 :rocket:

File: public/js/request.view.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    var headers = _.extend( baseHeaders, parsedHeaders );
+    return headers;
+  },
+  parseHeaders : function( raw ) {
+    var headers = {};
+    raw.split('\n').forEach(function( line ) {
+      var a = line.split(':', 2);
+      if (a[0] && a[1]) headers[a[0].trim()] = a[1].trim();
+    });
+    log.debug('headers:');
```
Comment (jasonmadigan) [inline public/js/request.view.js:109]: `log.debug('headers:', headers)` maybe?

**Comment by jasonmadigan** [comment]
@cianclarke already reviewed by @lfryc too, but gave it a scan too - looks good!

**Pull request merged**